pool/errors.go

```diff
@@ -41,4 +41,7 @@ var (
 
 	// ErrGasUintOverflow is returned when calculating gas usage.
 	ErrGasUintOverflow = errors.New("gas uint64 overflow")
+
+	// ErrIntrinsicGasPrice is returned if the transaction has specified lower gas price than the minimum allowed.
+	ErrIntrinsicGasPrice = errors.New("intrinsic gas price too low")
```

Comment:
```suggestion
	ErrGasPrice = errors.New("gas price too low")
```

We use intrinsic for errors from the rom that prevent the tx from executing (nonce, balance, signature, ...)